force-app/main/default/lwc/recurringDonationTable/recurringDonationTable.js

```diff
@@ -0,0 +1,93 @@
+import { LightningElement, api } from 'lwc';
+import commonAmount from '@salesforce/label/c.commonAmount';
+import commonPaymentMethod from '@salesforce/label/c.commonPaymentMethod';
+import RDCL_Frequency from '@salesforce/label/c.RDCL_Frequency';
+import lblStatus from '@salesforce/label/c.lblStatus';
+import dateStarted from '@salesforce/label/c.dateStarted'
+import lastDonationDate from '@salesforce/label/c.lastDonationDate';
+import nextInstallment from '@salesforce/label/c.nextInstallment';
+import lastUpdate from '@salesforce/label/c.lastUpdate';
+import commonActions from '@salesforce/label/c.commonActions';
+
+export default class RecurringDonationTable extends LightningElement {
+    @api
+    donationTypeFilter = 'all';
+    data = [{
```

Comment:
@maurimart Add some string values to the "Next Installment" mock data, fix date format.